lib/quark/compose.ex

```diff
@@ -73,7 +73,7 @@ defmodule Quark.Compose do
   def g <|> f, do: compose(g, f)
 
   @doc ~S"""
-  Function composition, from the back of the lift to the front
+  Function composition, from the back of the list to the front
```

Comment:
I was trying to go for small changes. I feel like introducing so many words like "pipes" and "arguments", "first" and "second" etc add confusion to the sentence.
```
Function composition, pipes a value to the first argument then that output is piped to the second argument.
```
Perhaps we're trying to say:
```
Function composition, from left to right
```
Assuming some basic knowledge of what it means to compose two functions, I would have thought most people would understand this?